### PR TITLE
middleman: add page

### PR DESCRIPTION
**Pull request by hasantezcan**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

This my first contributing in tldr community. I am using tldr for a while time. Today I notice middleman page does not exist on the tldr database. I try to add them very basicly. I hope it will be helpful for other peoples.

### CODE REVIEW

File: pages/common/middleman.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# middleman
+
+> Static site generator written in Ruby.
+> More information: <https://middlemanapp.com/>.
+
+- Create a new middleman project:
```
Comment (zdroid) [inline pages/common/middleman.md]:
```suggestion
- Create a new Middleman project:
```

It should be in upper-case according to the website.

File: pages/common/middleman.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start local server for current project on a specified port:
+
+`middleman server -p "{{port}}"`
+
+- Build your project to prepare to deploy:
+
+`bundle exec middleman build`
+
+- Deploy your middleman project:
```
Comment (zdroid) [inline pages/common/middleman.md]:
```suggestion
- Deploy your Middleman project:
```

**Comment by hasantezcan** [comment]
I updated this PR. it's working properly right now.

**Comment by hasantezcan** [comment]
Why this PR isn't merge yet?

**Comment by zdroid** [comment]
It isn't? :smile:

**Comment by hasantezcan** [comment]
finally :smile:

File: pages/common/middleman.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Start local server for current project on port 4567:
+
+`middleman server`
+
+- Start local server for current project on a specified port:
+
+`middleman server -p "{{port}}"`
+
+- Build your project to prepare to deploy:
```
Comment (sbrl) [inline pages/common/middleman.md]: Perhaps we could avoid the use of `your`  here? Generally we try and avoid it in tldr pages. What about this instead?

```suggestion
- Build the project in the current directory to prepare for deployment:
```

File: pages/common/middleman.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`bundle exec middleman build`
+
+- Deploy your Middleman project:
```
Comment (sbrl) [inline pages/common/middleman.md]: Same here.

```suggestion
- Deploy the Middleman project in the current directory:
```

File: pages/common/middleman.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# middleman
+
+> Static site generator written in Ruby.
+> More information: <https://middlemanapp.com/>.
+
+- Install the Middleman gem:
```
Comment (agnivade) [inline pages/common/middleman.md]: That's not a middleman command. Let's remove this.

Comment (hasantezcan) [reply pages/common/middleman.md:6]: You are right. I am fixing that.

File: pages/common/middleman.md (hunk 3 above)
Comment (hasantezcan) [reply pages/common/middleman.md:22]: Fixed :+1:

File: pages/common/middleman.md (hunk 4 above)
Comment (hasantezcan) [reply pages/common/middleman.md:26]: Fixed :+1:

**Pull request merged**